stream-parent/stream-receiver/src/main/java/org/interledger/stream/receiver/StatelessStreamReceiver.java

```diff
@@ -14,12 +14,7 @@
 import org.interledger.stream.StreamPacket;
 import org.interledger.stream.StreamUtils;
 import org.interledger.stream.crypto.StreamEncryptionService;
-import org.interledger.stream.frames.ConnectionCloseFrame;
-import org.interledger.stream.frames.ErrorCode;
-import org.interledger.stream.frames.StreamFrame;
-import org.interledger.stream.frames.StreamFrameType;
-import org.interledger.stream.frames.StreamMoneyFrame;
-import org.interledger.stream.frames.StreamMoneyMaxFrame;
+import org.interledger.stream.frames.*;
```

Comment:
Checkstyle will complain about the star import. I disabled star imports in IDEA to avoid this.